api/HTMLScriptElement.json

```diff
@@ -409,6 +409,54 @@
           }
         }
       },
+      "referrerPolicy": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLScriptElement/referrerPolicy",
+          "support": {
+            "chrome": {
+              "version_added": true
```

Comment:
I've been trying to learn my way around the Chromium stuff and I think actually found this one in version 70: https://storage.googleapis.com/chromium-find-releases-static/bb8.html#bb89c0c9a58544f718fb06f352282fa13411245e

Reply:
Thanks Daniel! Updated.